.github/workflows/tests-gpu.yml

```diff
@@ -29,7 +29,6 @@ jobs:
         github.event_name == 'push' ||
         (
           github.event_name == 'pull_request' &&
-          contains(github.event.pull_request.labels.*.name, 'gpu') &&
```

Comment:
We shouldn't remove this. We want to use this as a way to programmatically enable the GPU tests by adding the label "gpu" to the PR.

Reply:
![image](https://user-images.githubusercontent.com/858615/189157682-73920d01-af2f-4ecf-9fdb-e5da523e3ed2.png)

After adding the label the GPU tests should run (I just added it now).

Reply:
Ah I see 😄